src/main/java/bio/terra/cloudfunctions/common/EventWrapper.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.cloudfunctions.common;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class EventWrapper {
+  /**
+   * Parse an event String and returns the event represented by generic event type E.
+   *
+   * @param json the event String
+   * @param type the type of the event
+   * @return object representing event type E
+   */
+  public static <E> E parseEvent(String json, Class<E> type) {
```

Comment:
This is a useful method, but the code is not necessarily specific to events.  What about something like and putting that method in GsonWrapper perhaps or just a GsonConverter or GsonUtils class?
```
    public static <T> T convertFromClass(String s, Class<T> classOfT) {
    return GsonWrapper.getInstance().fromJson(s, classOfT);
  }
```

Reply:
Made a similar observation and commented on GsonConveter in your PR [qa-1518-drew](https://github.com/DataBiosphere/terra-gcs-bq-streaming-functions/pull/7).

The methods `public static <E> E parseEvent(String json, Class<E> type)` and `public static <T> T convertFromClass(String s, Class<T> classOfT)` have identical signatures so I am good in consolidating them into just one class: `GsonWrapper`, `GsonConverter`, or a common `GsonUtils` would be fine.